qcore/timeseries.py

```diff
@@ -57,6 +57,8 @@ def bwfilter(data, dt, freq, band, match_powersb=True):
     if match_powersb:
         if band == "highpass":
             freq *= 0.8956803352330285
+        elif band == "bandpass":
+            freq = np.asarray((freq*0.8956803352330285, freq*1.1164697500474103))
```

Comment:
Should we make them variables? I find it slightly uncomfortable to see repeating hard-coded constants.

Reply:
I think we should. Especially considering we're using the same number in two places.

Reply:
Generally I agree with that, but this is simple enough that I'm happy to leave this as technical debt that potentially never gets resolved

Reply:
And additionally this is functionality that probably won't be used very often. :upside_down_face: